src/explore-education-statistics-ckeditor/plugins/comments/AddCommentPlaceholderCommand.js

```diff
@@ -25,10 +25,13 @@ export default class AddCommentPlaceholderCommand extends Command {
     const range = editor.model.document.selection.getFirstRange();
 
     editor.model.change(writer => {
+      if (editor.model.markers.has('commentplaceholder')) {
```

Comment:
Might be a good idea to extract the `commentplaceholder`, `comment` and `resolvedcomment` strings into their own module and import them in across the plugin files to make refactoring easier.

Reply:
I've moved them to a new constants file.

Reply:
Cool, unfortunately, I can't seem to see the file in this PR?